website/docs: endpoint devices (#18634)
goauthentik/authentik: website/docs/endpoint-devices/device-authentication/ssh-authentication.mdx

---
title: SSH authentication
sidebar_label: SSH authentication
tags: [ssh, authentik Agent]
---

> You can use the [authentik agent](../authentik-agent/index.mdx) to authenticate SSH connections using authentik credentials between endpoint devices.

Reviewer: ```suggestion
You can use the [authentik Agent](../authentik-agent/index.mdx) to authenticate SSH connections ubetween endpoint devices using authentik credentials.
```

> Currently, this is only supported when the SSH endpoint is a [Linux](../authentik-agent/agent-deployment/linux.md) device, see the [Configure SSH authentication on an endpoint device](#configure-ssh-authentication-on-an-endpoint-device) section for more details.

Reviewer: ```suggestion
Currently, only [Linux](../authentik-agent/agent-deployment/linux.md) devices can serve as SSH endpoints. See  [Configure SSH authentication on an endpoint device](#configure-ssh-authentication-on-an-endpoint-device) section for more details.
```
